Update User_Guide.rst (#357)
pyansys/PyAEDT: doc/source/Resources/User_Guide.rst

User Guide

> ----------

Reviewer: Remove the following section but keep the "User Guide" section since we need this to get the top level titles to display "User Guide"

> PyAEDT Overview
> ---------------

Reviewer: ```suggestion
```

    # Desktop is automatically released here.

The previous command launches AEDT and initializes a new Circuit design.

.. image:: ./aedt_first_page.png
  :width: 800
  :alt: Electronics Desktop Launched

> The same result can be obtained with the following code.

Reviewer: I'd prefer to keep the colon, but it's up to you.

> The same result can be obtained with the following code.

Author: ```suggestion
The same result can be obtained with the following code:
```

Variables
~~~~~~~~~

> PyAEDT provides for easily getting and setting variables inside a project or a design.

Reviewer: ```suggestion
PyAEDT provides a simplified interface getting and setting variables inside a project or a design.
```

> PyAEDT provides for easily getting and setting variables inside a project or a design.

Author: ```suggestion
PyAEDT provides a simplified interface for getting and setting variables inside a project or a design.
```

You simply need to initialize a variable as a dictionary key. If you use ``$`` as the prefix 

> for the variable name, a project variable is created.

Reviewer: ```suggestion
for the variable name, a project-wide variable is created.
```

Modeler
~~~~~~~
Object-oriented programming is used to create and manage objects in the AEDT 3D and 2D Modelers. 
You can create an object and change properties using getters and setters.

> 

Reviewer: ```suggestion
```
